corehq/apps/style/static/app_manager/less/preview_app-main.less

```diff
@@ -166,14 +166,16 @@ preview_app/less */
 }
 
 .btn-preview-refresh::after {
-  content: '✔';
-  color: green;
+  content: "\f00c";
```

Comment:
it uses the font awesome checkmark icon instead. keeps things standardized. the other checkmark changes depending on the system and just looks awkward against the other icons